appcli/commands/service_cli.py

```diff
@@ -239,6 +256,11 @@ def _action_orchestrator(
             pre_hook = hooks.pre_shutdown
             post_hook = hooks.post_shutdown
 
+        elif action == ServiceAction.STATUS:
+            action_run_function = self.orchestrator.status
```

Comment:
Do we want to consider writing pre/post hooks for `STATUS` calls?

Reply:
Nah - it's just a snapshot look at what's running, not a lifecycle event